awslimits/settings.py

```diff
@@ -7,16 +8,18 @@
 WTF_CSRF_SECRET_KET = SECRET_KEY
 
 ROLE_ARN = os.environ.get("ROLE_ARN")
+ACCOUNT_ID = re.findall('\d+', ROLE_ARN)[0]
+ACCOUNT_ROLE = ROLE_ARN.split('/')[-1]
 REGION_NAME = os.environ.get("REGION_NAME", "us-east-1")
+PREMIUM_ACCOUNT = int(os.environ.get("PREMIUM_ACCOUNT", 0))
```

Comment:
I'd go and default to `1` (I just asked our account manager to set up enterprise support to the internal tools account).